.all-contributorsrc

```diff
@@ -1,7 +1,7 @@
 {
   "files": [
     "docs/README.md",
-    "docs/CONTRIBUTORS.md"
+    "Phonebook.Frontend/src/assets/CONTRIBUTORS.md"
```

Comment:
Willst du die Datei in `docs/CONTRIBUTERS.md` löschen, wenn sie nicht mehr aktualisiert wird?

Reply:
hatte es vergessen zu committen 🥇

Reply:
es ist immer noch nicht gelöscht